### ISSUE TITLE
PPLNS

### ISSUE THREAD

**Issue by yrjo** [SadBatman/cryptonote-sumokoin-pool#32]
Hi,
which payment scheme is set by default?
Is there any way how to set up PPLNS?
thanks a lot in advance!
cheers

**Comment by mrtiritas** [comment]
There's no way right now i think.

But it should be an option. The "other" pool software, nodejs-pool has it, and it's in some of the major pools.

PPLNS should be a new feature, if more people request it, and the devs agree.

**Comment by Ossii** [comment]
@mrtiritas Is there any forks of that around made for sumokoin?

**Comment by MrKitai** [comment]
It's not really polite to mention another software capabilities in a GIT. Just did it to ask for the feature.

Won't mention anything else about that software ;-)

**Comment by javierin** [comment]
I agree PPLNS would be a nice enhancement and would decrease pool hopping. +1 to the feat request

**Comment by Bathmat** [comment]
I wanted to say +1 for PPLNS.

I just set up my own pool using your code,  and I wanted to say thanks! I had literally zero coding, linux/ubuntu, html experience before I did this, and your guide made creating the pool really easy.

I think adding the PPLNS feature would help strengthen the Sumo network by being able to attract more miners to the smaller pools and increasing decentralization.
